static/src/js/components/CollectionDetails/CollectionDetailsBody.js

```diff
@@ -0,0 +1,253 @@
+                      const key = `temporal_entry_${i}`
+                      return <span key={key}>{entry}</span>
+                    })}
+                  </dd>
+                </>
+                )
+              }
+              <dt>GIBS Imagery Projection Availability:</dt>
+              <dd>
+                {gibsLayers && gibsLayers}
+              </dd>
+            </dl>
+          </div>
+          <div className="col collection-details-body__group--secondary">
+            <dl className="collection-details-body__info">
+              <dt>Science Keywords:</dt>
+              <dd>
+                {
+                  scienceKeywords.length === 0 && <span>Not Available</span>
+                }
+                {
+                  scienceKeywords.length > 0 && buildScienceKeywordList(scienceKeywords)
+                }
+              </dd>
+            </dl>
+          </div>
+        </div>
+        <div className="row collection-details-body__row">
+          {/* eslint-disable-next-line react/no-danger */}
+          <div className="col collection-details-body__summary" dangerouslySetInnerHTML={{ __html: summary }} />
+        </div>
+        <div className="row">
+          {
+            dataCenters.length && (
+              <ul className="col collection-details-body__provider-list">
+                {
+                  dataCenters.map((dataCenter, i) => {
+                    const key = `data_center_${i}`
+                    return (
+                      <CollectionDetailsDataCenter key={key} item={i} dataCenter={dataCenter} />
+                    )
+                  })
+                }
+              </ul>
+            )
+          }
+        </div>
+      </div>
+      {/* <div>
+        More Actions
+        <div>Handoff Links</div>
+      </div> */}
+      <CollapsePanel
+        className="collection-details-body__for-devs"
+        headerText="For Developers"
+        scrollToBottom
+      >
+        <div className="row">
+          <div className="col-auto">
+            <h4 className="collection-details-body__dev-cat">More Metadata</h4>
+            <ul className="collection-details-body__dev-list">
+              {urls.native && buildForDeveloperLink(urls.native)}
+              {urls.atom && buildForDeveloperLink(urls.atom)}
+              {urls.echo10 && buildForDeveloperLink(urls.echo10)}
+              {urls.iso19115 && buildForDeveloperLink(urls.iso19115)}
+              {urls.smap_iso && buildForDeveloperLink(urls.smap_iso)}
+              {urls.dif && buildForDeveloperLink(urls.dif)}
+            </ul>
+          </div>
+          <div className="col-auto">
+            <h4 className="collection-details-body__dev-cat">API Endpoints</h4>
+            <ul className="collection-details-body__dev-list">
+              {urls.granuleDatasource && buildForDeveloperLink(urls.granuleDatasource)}
+              {urls.osdd && buildForDeveloperLink(urls.osdd)}
+              {
+                // TODO: GIBS
```

Comment:
https://search.earthdata.nasa.gov/search/collection-details?p=C1219032685-LANCEMODIS&m=0.0703125!0!2!1!0!0%2C2&ff=Map%20Imagery has a GIBS link.

Reply:
And this one has _lots_ of GIBS links -> https://search.earthdata.nasa.gov/search/collection-details?p=C1426751946-LANCEMODIS&m=0.0703125!0!2!1!0!0%2C2&ff=Near%20Real%20Time

Reply:
OPeNDAP link -> https://search.earthdata.nasa.gov/search/collection-details?p=C191855458-LARC&m=0.0703125!0!2!1!0!0%2C2&q=mopit&ok=mopit

Reply:
MODAPS link -> https://search.earthdata.nasa.gov/search/collection-details?p=C190733713-LPDAAC_ECS&m=0.0703125!0!2!1!0!0%2C2&q=C190733713-LPDAAC_ECS&ok=C190733713-LPDAAC_ECS